data/migrations/README.md

```diff
@@ -0,0 +1,16 @@
+# Migrations for sovrin-node
+
+All migrations are python scripts with names satisfying the following rule:
+```
+<underscored_version-migration-should-be-applied-to>_<migration-name>
```

Comment:
@dhh1128 
1) I think we will have a single script almost always. yet I think we need to leave some room for us to wiggle, since it does not cost us anything.
2) It's actually the version you are coming from. I think it does not really matter what version you're updating to
3) Versions are underscored because they are python modules and a python module can not be dotted